administrator/components/com_finder/src/Model/FilterModel.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
-/**
+/**New chat
```

Comment:
```suggestion
/**
```
Revert that artifact added by whatever.